examples/test_tokenizer.cpp

```diff
@@ -27,7 +28,7 @@ void tokenizer_test(bert_ctx * ctx, const std::string& input, const std::vector<
         printf("]\n");
 
         for (size_t i = 0; i < result.size(); i++) {
-            bert_vocab_id a = expected[std::min(i, expected.size()-1)];
+            bert_vocab_id a = expected[min(i, expected.size()-1)];
```

Comment:
Why is std:: removed here? This doesn't compile on linux

Reply:
Can you test it ? I'll test it again for linux.

Reply:
fixed. Need a definition to avoid windows using another macro.